README.md

```diff
@@ -59,7 +59,7 @@ docker run  -d \
 			--volume ~/magic_mirror/config:/opt/magic_mirror/config \
 			--volume ~/magic_mirror/modules:/opt/magic_mirror/modules \
 			--name magic_mirror \
-			MichMich/MagicMirror
+			michmich/magicmirror
```

Comment:
@z0diak could you check if the dockerfile from [this commit](https://github.com/MichMich/MagicMirror/commit/6e33e063da3c83fd9bcffc89b72595933dc61d79) (#825) works?